__tests__/ship.test.js

```diff
@@ -4,20 +4,133 @@ describe("constructor", () => {
   it("returns an object", () => {
     expect(new Ship("Enterprise")).toBeInstanceOf(Object);
   });
-
+  const ship = new Ship("Enterprise");
   it("sets the name property", () => {
-    const ship = new Ship("Enterprise");
-
     expect(ship.name).toEqual("Enterprise");
   });
+  it("initial year of 0", () => {
+    expect(ship.yearsInService).toEqual(0);
+  });
+  it("initial fuel of 10", () => {
+    expect(ship.fuel).toEqual(20);
+  });
+  it("initial morale of 8", () => {
+    expect(ship.morale).toEqual(8);
+  });
 });
 
 describe("yearsInService", () => {
-  it("Increases year of service by 1", () => {
-    const ship = new Ship("Enterprise");
+  const ship = new Ship("Excelsior");
+  it("increments year by 1", () => {
+    ship.tourOfDuty();
+    expect(ship.yearsInService).toEqual(1);
+  });
+});
+
+describe("fuel", () => {
+  const ship = new Ship("Yorktown");
+  it("decrease fuel by 1", () => {
+    ship.tourOfDuty();
+
+    expect(ship.fuel).toEqual(19);
```

Comment:
Quite often its nice to see a line clear after the test setup and before the assertions as youve done here. Whether or not you take this approach is a coding standards thing usually dictated by your workplace but whichever way you go consistency across the whole project is always the way forward